values-changes.yaml

```diff
@@ -121,4 +121,6 @@ changes:
       - 'teamConfig.{team}.selfService.team.downloadDockerConfig': 'teamConfig.{team}.selfService.access'
   - version: 19
     deletions:
+      - 'apps.loki.storage.gcs.project'
+      - 'apps.loki.storage.gcs.instance'
```

Comment:
Should we add these deletions as `version: 20`? It seems that version 19 has already been released. 
[v2.9.0](https://github.com/redkubes/otomi-core/compare/v2.8.0...v2.9.0)

Reply:
Yes, we should merge in `main` once more.